scripts/vet.sh

```diff
@@ -197,7 +197,8 @@ GetSafeRegexMatch
 GetSuffixMatch
 GetTlsCertificateCertificateProviderInstance
 GetValidationContextCertificateProviderInstance
-XXXXX PleaseIgnoreUnused'
+XXXXX PleaseIgnoreUnused
+NewAddress is deprecated:'
```

Comment:
```suggestion
NewAddress is deprecated:
XXXXX PleaseIgnoreUnused'
```

That last line is there to avoid need to move the `'` around.  Also, let's try to keep this list sorted, so actually, please move this to right before `NewSubConn is deprecated:`.